gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/cockpit/command/handler/OrganizationCommandHandler.java

```diff
@@ -55,6 +57,8 @@ public OrganizationCommandHandler(OrganizationService organizationService) {
         OrganizationPayload organizationPayload = command.getPayload();
 
         try {
+            ExecutionContext executionContext = new ExecutionContext(organizationPayload.getId(), null);
```

Comment:
Can we ensure that subsequent processes never call the ExceutionContext.getEnvironmentId() ?
Or it will lead to an EnvironmentNotFoundException cause environementId is null

Reply:
it will lead to an EnvironmentNotFoundException cause environementId is null